lang/python/python3-speedtest-cli/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-speedtest-cli
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=speedtest-cli
+PKG_HASH:=cf1d386222f94c324e3125ba9a0d187e46d4a13dca08c023bdb9a23096be2e54
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/speedtest-cli-$(PKG_VERSION)
```

Comment:
Is explicitly setting `PKG_BUILD_DIR` here necessary?

Reply:
Yes It's not needed. Resolved.